handler/openid/validator.go

```diff
@@ -38,6 +39,7 @@ import (
 type OpenIDConnectRequestValidator struct {
 	AllowedPrompt []string
 	Strategy      jwt.JWTStrategy
+	IsRedirectURISecure func(*url.URL) bool
```

Comment:
Adding this for completeness. We are allowing configuring `IsRedirectURISecure` in some places, but not in all. I added it here so that we can configure it here as well.